routes/api/selectors.js

```diff
@@ -7,16 +7,50 @@ router.get('/', function(req, res, next) {
 
```

Comment:
Ok. When I used sequelize, it allows for rails like db/seed and db/migration. Does that exist in gcloud or do you have to build that manually?